docs/design/3_1_6_base_station.tex

```diff
@@ -1 +1,40 @@
-TODO: How many base stations? What will base station look like? How are machine learning results displayed?
\ No newline at end of file
+The base station in the system is a mobile device that connects to the on-board computing platform, it is used for:
+\begin{itemize}
+    \item monitoring the status of the system.
+    \item verifying the ML result.
+    \item storing the log data and video for further analysis.
+    \item control the computing platform, i.e. start object detection.
+\end{itemize}
+
+\textbf{Options for the Base Station}
+
+First option is a laptop. A laptop is a portable personal computer which has the following features:
```

Comment:
I don't think this list is necessary - if the reader has gotten this far I think they know what a laptop is :P

Perhaps just discuss why a laptop would be advantageous over a mobile device in a short section.